modules/02_welfare/utilitarian/equations.gms

```diff
@@ -85,8 +85,7 @@ q02_inconvPenCoalSolids(t,regi)$(t.val > 2005)..
     v02_inconvPenCoalSolids(t,regi)
   =g=
 *' local air pollution for coal: inconvinience penalty applies only for buildings use; slack variable ensures that v02_inconvPen can stay > 0 
-    p02_inconvpen_lap(t,regi,"coaltr") * (vm_prodSe(t,regi,"pecoal","sesofos","coaltr") 
-  - (vm_cesIO(t,regi,"fesoi") + pm_cesdata(t,regi,"fesoi","offset_quantity")))
```

Comment:
Should this rather be a sum `fesoi ≡ feso_cement + feso_chemicals + feso_steel + feso_otherIn?`

Reply:
ahhhhh. 

I do have an old branch lying around where I fixed this wrong formulation. 
Let me check if I find it again....

Reply:
@JakobBD Are you aware of this? Looks like air pollution inconvenience penalty is reduced by industry and only applied for buildings. Can we delete this, or do we have to rewrite the term?

Reply:
https://github.com/remindmodel/remind/pull/1953

will add the scenario results next Monday

Reply:
solved with this PR: https://github.com/remindmodel/remind/pull/1953#pullrequestreview-2561691726